workflow/resources/config/default.yaml

```
@@ -231,6 +233,21 @@ calling:
         labels:
           clinical significance: pathogenic, risk factor, or drug response
         subcategory: known somatic variants
+      
+      somatic_splice_mutation:
+        varlociraptor:
+          - somatic_tumor_high
+          - somatic_tumor_low
+        filter:
+          - splice_mutation
```

```suggestion
          - splice_mutation
          - revel_malign
          - novel
```
The novel filter should be implemented as a result of splitting up the impact_high_novel filter, and accordingly used in the other callsets where now impact_high_novel is used.